Pages/Incoming.cshtml.cs

```diff
@@ -96,15 +96,11 @@ private async Task<CompareResult> GetCommitsBehindAsync(GitHubInfo gitHubInfo, B
 
         private async Task<(int?, DateTimeOffset)> GetCommitInfo(GitHubInfo gitHubInfo, Build build)
         {
-            var commitAge = DateTimeOffset.Now;
+            var commitAge = build.DateProduced;
             int? commitDistance = null;
-            if (gitHubInfo == null)
+            var comparison = await GetCommitsBehindAsync(gitHubInfo, build);

```

Comment:
You still need to check if `gitHubInfo` is null

Reply:
Fixed.